### ISSUE TITLE
Cannot be installed in an environment with Python 3.9

### ISSUE THREAD

**Issue by DragonEggBedrockBreaking** [spyder-ide/spyder-terminal#251]
## Python cannot be installed on a conda environment with Python 3.9

**What steps will reproduce the problem?**

1. Download Conda (Anaconda/Miniconda)
2. Create a python 3.9(.2) environment and activate it
3. Run `conda install spyder-terminal -c spyder-ide`

**What is the expected output? What do you see instead?**
I expect spyder-terminal to successfully install. I instead see the following error:

```
Specifications:

  - spyder-terminal -> python[version='3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to.
```

## Versions and main components

* Terminal Version: Conda 4.9.2
* Spyder Version: Spyder 4.2.5
* Python Version: Python 3.9.2
* Operating system: Windows 10

**Comment by steff456** [comment]
Hi @DragonEggBedrockBreaking,

This error pops out in a new and clean environment or did you installed something before in it?

**Comment by DragonEggBedrockBreaking** [comment]
> Hi @DragonEggBedrockBreaking,
> 
> This error pops out in a new and clean environment or did you installed something before in it?

I think it was clean, but I don’t remember. I will test tomorrow (on python 3.9.4) on a new clean environment.

**Comment by steff456** [comment]
Let me know :)

**Comment by DragonEggBedrockBreaking** [comment]
I installed miniconda (python 3.9), created a fresh environment, followed the commands in the releases page:
```
conda update conda
conda install spyder
conda install spyder-terminal -c spyder-ide
```
and still got this error after running the third command (first two were successful):
```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - spyder-terminal -> python[version='3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

- Conda 4.10.1
- Spyder 5.0.0
- Python 3.9.4
- Windows 10

**Comment by steff456** [comment]
This error is happening because you are installing Spyder 5.0.0. Right now spyder-terminal doesn't support the new API so it is not working as expected. Please try to install Spyder 4 in the new environment and everything should work fine. I have the update of this plugin in my priority list, so I hope to be working in the migration soon.

Please let me know if Spyder 4 is back to live :)

**Comment by DragonEggBedrockBreaking** [comment]
How do I install Spyder 4? What command do I need?

**Comment by steff456** [comment]
If you are using conda please create a new environment and pin Spyder version,

```
conda create -n test python=3.9
conda install spyder=4.2.5
conda install spyder-terminal
spyder
```

That should install you the last release of Spyder 4 with Spyder-terminal and it should be working.

**Comment by DragonEggBedrockBreaking** [comment]
I ran successfully:
```
conda create -n test python=3.9
conda activate test
conda install spyder=4.2.5
```

I then ran the command `conda install spyder-terminal` and I got an error:
```
(test) C:\Users\user>conda install spyder-terminal
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - spyder-terminal

Current channels:

  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

I also ran a different command which is the command stated [on the releases page](https://github.com/spyder-ide/spyder-terminal/releases) - `conda install spyder-terminal -c spyder-ide`, and I got a different error:
```
(test) C:\Users\user>conda install spyder-terminal -c spyder-ide
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: |
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - spyder-terminal -> python[version='3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.8,<3.9.0a0|>=3.7,<3.8.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```
(this second error is the same error as I was getting before with Spyder 5.0.0)

**Comment by ccordoba12** [comment]
It seems we don't have Spyder-terminal packages for Python 3.9, but that will be fixed in our next version, to be released early next month.

**Comment by steff456** [comment]
Hi @DragonEggBedrockBreaking,

We just released spyder-terminal 1.0.0 available for Python 3.9 in pip and in conda-forge channel. Please take into account that you need to install Spyder 5.0.5 to use it. 

Thanks for reporting!
